ProcessLib/ThermoHydroMechanics/Tests.cmake

```diff
@@ -0,0 +1,81 @@
+# ThermoHydroMechanics; Small deformation, linear poroelastic, homogeneous
+AddTest(
+    NAME ThermoHydroMechanics_square_1e0
+    PATH ThermoHydroMechanics/Linear/Square_sealed_homogeneous
+    EXECUTABLE ogs
+    EXECUTABLE_ARGS square_1e0.prj
+    WRAPPER time
+    TESTER vtkdiff
+    REQUIREMENTS NOT OGS_USE_MPI
+    DIFF_DATA
+    expected_square_1e0_pcs_0_ts_10_t_1000.000000.vtu square_1e0_pcs_0_ts_10_t_1000.000000.vtu displacement displacement 1e-8 1e-8
```

Comment:
For two linear displacements in x and y directions the error margin is too generous. |u| is around 1e3 and linear, I'd expect an absolute error of around 1e-12 and because the displacement is 0 in some places the relative error does not make much sense (there) and should be set to zero.
The remark includes all other primary variables in this test.

Reply:
It is not clear to me what properties of a non-linear process are being tested with this benchmark on a single quad element. All values are set via boundary conditions.

Reply:
This test comes from OGS5. There is a same test implemented in OGS5 for the IGLU project to verify the THM process in OGS5. The results are the same in OGS5 and OGS6 (0.1042MPa). I will try to see whether the 1e-3 can be extended to 1e-12.

Reply:
I have checked the analytical and numerical solution again carefully. From the analytical solution, the result is 0.10415999999999997 MPa and from the numerical solution, it is 0.10416 MPa. The absolute error is less than 1e-12 MPa.

Reply:
Then you should reflect this in the ctests.

Reply:
The corresponding ctests are updated.